packages/e2e-test-utils/README.md

```diff
@@ -360,6 +360,33 @@ _Returns_
 
 -   `Promise`: Promise that uses `mockCheck` to see if a request should be mocked with `mock`, and optionally transforms the response with `responseObjectTransform`.
 
+<a name="multiSelectBlocksByIds" href="#multiSelectBlocksByIds">#</a> **multiSelectBlocksByIds**
+
+Select multiple blocks on the editor given the start clientId and the end clientId.
+First selects the start block in order to properly multi select.
+
+_Parameters_
+
+-   _start_ `string`: Identifier of the start block.
+-   _end_ `string`: Identifier of the end block.
+
+<a name="multiSelectBlocksByRange" href="#multiSelectBlocksByRange">#</a> **multiSelectBlocksByRange**
+
+Select multiple blocks on the editor given the start block order
+and the total amount of wanted selected blocks.
+Validates if not available blocks for selection.
+
+Also selects the start block in order to properly multi select.
+
+_Usage_
+
+// selects three blocks -- from the second block to fourth (2, 3, 4).
```

Comment:
Something is wrong with the example in JSDoc.

Reply:
Thanks! Fixed it.